cron/polygon/standalone-runner.js

```diff
@@ -0,0 +1,252 @@
+const hre = require("hardhat");
+const fs = require('fs');
+
+const IERC20Abi = require("./abi/IERC20Upgradeable.json");
+const IControllerV1Abi = require("./abi/Controller.json");
+const vaultAbi = require("./abi/Vault.json");
+const { web3 } = require("hardhat");
+const settings = require("./settings.json");
+
+// logic control
+const nextVault = require("./next-vault.json");
+const vaultDecision = require("./vault-decision.json");
+const { randomInt } = require("crypto");
+
+const ethers = hre.ethers;
+
+// Only execute the `doHardWork` when
+// the profit share is `greatDealRatio` times better than the gas cost in Ether
+const greatDealRatio = 6;
+// or when the funds available for invest is 1/(idleFraction) of the total funds.
+// note that funds that are available for invest are different from funds sitting in vault
+const idleFraction = 20;
+const bedBot = "0xbed04c43e74150794f2ff5b62b4f73820edaf661";
+
+const addresses = require('../../../harvest-api/data/mainnet/addresses.json').MATIC;
+const allVaults = Object.keys(addresses);
```

Comment:
https://github.com/harvest-finance/harvest-api/blob/master/data/mainnet/addresses.json#L1538 contains many addresses not only vaults. Can you re-organize `addresses.json`'s MATIC section so that there is a subsection for vaults? You can do the same as the section for mainnet: `V2` contains vaults only.

Reply:
added V2